compiler/rustc_codegen_llvm/src/back/lto.rs

```diff
@@ -53,8 +71,13 @@ fn prepare_lto(
         Lto::No => panic!("didn't request LTO but we're doing LTO"),
     };
 
+    let llvm_reserved_symbols = get_llvm_reserved_symbols();
```

Comment:
again, I think this variable should be named `llvm_preserved_symbols`, not `llvm_reserved_symbols`, if I understand this correctly...